addons/binding/org.openhab.binding.bigassfan/src/main/java/org/openhab/binding/bigassfan/handler/BigAssFanHandler.java

```diff
@@ -483,6 +461,17 @@ private boolean isLightPresent() {
         return false;
     }
 
+    private static final StringType LIGHT_COLOR = new StringType("COLOR");
+
+    private boolean isLightColor() {
+        if (fanStateMap.containsKey(CHANNEL_LIGHT_COLOR)) {
```

Comment:
This is the same as:

```java
return fanStateMap.containsKey(CHANNEL_LIGHT_COLOR) && LIGHT_COLOR.equals(fanStateMap.get(CHANNEL_LIGHT_COLOR))
```

Note that I also inverted the equals to make it more null-safe

Reply:
> Note that I also inverted the equals to make it more null-safe

Good catch. Thanks.